### PR TITLE
Feature/redesign app

### PR DESCRIPTION
**Pull request by OSKyriienko**

### CODE REVIEW

File: src/components/leaves/Leaves.jsx [hunk 1]
```diff
@@ -1,337 +1,33 @@
...
+    <div className="nd-leaves-wrapper">
+      <Tabs
+        activeKey={activeTab}
+        defaultActiveKey="userLeave"
+        onChange={tab => setCurrentTab(tab)}
+      >
+        <TabPane tab="Personal" key="userLeaves">
+          {activeTab === 'userLeaves' && <LeavesUser />}
+        </TabPane>
+        <TabPane tab={localStorage.getItem('currentWs')} key="teamLeaves">
```
[the remark this answers is not in the document]
Comment (OSKyriienko) [reply src/components/leaves/Leaves.jsx]: It's the same as profileService.currentWs.name. Fixed.

File: src/components/leaves/Leaves.jsx [hunk 2]
```diff
@@ -1,337 +1,33 @@
...
-      {visible && createLeaveRequest()}
-      {listLeaves(currentUser, 'Leaves')}
-      {profileService.owner.userId === currentUser[0].userId &&
-        listLeaves(users, 'Team Leaves')}
-    </>
+    <div className="nd-leaves-wrapper">
+      <Tabs
+        activeKey={activeTab}
+        defaultActiveKey="userLeave"
+        onChange={tab => setCurrentTab(tab)}
```
[the remark this answers is not in the document]
Comment (OSKyriienko) [reply src/components/leaves/Leaves.jsx]: No need. Fixed.

File: src/components/leaves/LeavesUser.jsx [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+                <div>
+                  <div>
+                    <label>Type of Leave</label>
+                    <Field
+                      name="type"
+                      component={SelectForm}
+                      placeholder="Choose Type of Leave"
+                      options={leaveTypes}
+                    />
+                    <br />
```
[the remark this answers is not in the document]
Comment (OSKyriienko) [reply src/components/leaves/LeavesUser.jsx]: Just for interval between fields. Fixed by using class.

File: src/components/leaves/LeavesTeam.jsx [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+      }
+      setLoading(false);
+    })();
+  }, []);
+
+  const userLeaves = user => {
+    const data = vacations
+      .filter(item => item.userId === Number(user.id))
+      .map(item =>
+        assign(
```
[the remark this answers is not in the document]
Comment (OSKyriienko) [reply src/components/leaves/LeavesTeam.jsx]: Redundant. fixed.

File: src/components/leaves/LeavesUser.jsx [hunk 5]
```diff
@@ -0,0 +1,346 @@
...
+      setLoading(false);
+    })();
+  }, []);
+
+  const onSubmitCreateLeaves = async data => {
+    setLoading(true);
+    let type;
+    switch (data.type) {
+      case 'Paid vacation':
+        type = 'VACATION_PAID';
```
Comment (marcusn90) [inline src/components/leaves/LeavesUser.jsx]: Move vacation types to constants

**Pull request merged**